app/d3/multi-line-chart.js

```diff
@@ -167,6 +168,10 @@ angular.module('katGui.d3')
                             scope.options.yAxisValues = [];
                         }
                         d.value = String(d.value);
+                        if(!d.value) {
```

Comment:
Suggest a space is added for consistency
```suggestion
                        if (!d.value) {
```